specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/examples/DdosCustomPolicyCreate.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Please put this file in `Microsoft.Network/stable/2018-10-01`, not `2018-08-01` (same comment for other example jsons)

Reply:
done

Reply:
You should change folder where you put this file in the ```2018-10-01``` folder

Reply:
done